newscoop/src/Newscoop/GimmeBundle/Controller/ArticlesListController.php

```diff
@@ -141,17 +144,13 @@ public function getPlaylistsArticlesAction(Request $request, $id)
         }
 
         $playlistArticles = $em->getRepository('Newscoop\Entity\Playlist')
-            ->articles($playlist, null, false, null, null, $onlyPublished, true)->getResult();
+            ->articles($playlist, array(), true, null, null, $onlyPublished, true)->getResult();
 
-        $articlesIds = array();
+        $articles = array();
         foreach ($playlistArticles as $playlistArticle) {
-            $articlesIds[] = $playlistArticle['articleId'];
+            $articles[] = $playlistArticle->getArticle();
```

Comment:
No, it wont fail. I do the `getResult()` (see lines above) so it returns an array of `PlaylistArticle` objects from which I take the `Article` object using `$playlistArticle->getArticle()`